docs/identity-api-reference.md

```
@@ -0,0 +1,796 @@
+- Redis is used for replay prevention
+
+---
+
+#### POST `/connect/token`
+
+OpenIddict token endpoint for refresh token exchange.
+
+**Authentication**: None
+
+**Request Body** (form-urlencoded):
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)